### ISSUE TITLE
Example with curl

### ISSUE THREAD

**Issue by chasset** [TfLlu/TfL-API-Client#2]
Hello,

Could you provide an example in bash?

The following example doesn't work.
```
curl https://api.tfl.lu/v1/stream/
```
It leads to a message error : 'Transport unknown'.

Even this one:
```
curl https://api.tfl.lu/v1/stream/BikePoint
```

Best regards,

Pierre-Olivier

**Comment by dattn** [comment]
The streaming API will not work with curl.
You need to use a socket.io-client.
I'ill try to fix our client in node mode (Issue #1)

**Comment by chasset** [comment]
I have an example using browerify and browser-run. It adds 2 dependencies, but it works. Tell me if you are interested
